### DISCUSSION TITLE
Transfer Learning Classification with MMPretrain

### DISCUSSION THREAD

**Discussion by christian-holland** [open-mmlab/mmclassification#1669] [Q&A] [answered]
Please correct me if I am missing something, but as far as I understood training "custom" datasets by using the given configs does not use any pretrained weights (e.g. for Resnet18), as this repo is meant to do the actual pretraining.

Now if I instead do want to do some classic finetuning for classification tasks, using pretrained weights, is there some recommendation how to approach this task with this repo?

**Comment by Ezra-Yu** [comment, marked as answer]
`how to ft` refer to this [tutorial](https://mmpretrain.readthedocs.io/en/latest/notes/finetune_custom_dataset.html).


as the example:
```
_base_ = [
    '../_base_/models/resnet50.py',           # model settings
    '../_base_/datasets/imagenet_bs32.py',    # data settings
    '../_base_/schedules/imagenet_bs256.py',  # schedule settings
    '../_base_/default_runtime.py',           # runtime settings
]

# >>>>>>>>>>>>>>> Override model settings here >>>>>>>>>>>>>>>>>>>
model = dict(
    backbone=dict(
        frozen_stages=2,
        init_cfg=dict(
            type='Pretrained',
            checkpoint='https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_8xb32_in1k_20210831-ea4938fc.pth',
            prefix='backbone',
        )),
    head=dict(num_classes=10),
)
# <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<\
```

**model.backbone.init_cfg = dict(type='Pretrained', checkpoint="path_or_url", prefix='backbone',)** do the load pretrained weight configuration.
